### ISSUE TITLE
  (video toggling ON and OFF is seen intermittently) when 2nd attendee join video call , and start sharing his video attendee video blinking

### ISSUE THREAD

**Issue by kapilmyakal** [aws/amazon-chime-sdk-component-library-react#744]
### What happened and what did you expect to happen?

ConsoleLogger.ts:82 2022-02-11T10:36:11.229Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b was canceled due to 
ConsoleLogger.ts:82 2022-02-11T10:36:08.102Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/30263450-fbe61091383fd6b/Timeout15000ms/UpdateSession/SetLocalDescriptionTask error: Failed to execute 'setLocalDescription' on 'RTCPeerConnection': Failed to set local offer sdp: Failed to set local video description recv parameters for m-section with mid='1'.
log @ ConsoleLogger.ts:82
index.esm.js:7005 [MeetingManager audioVideoDidStart] Meeting started successfully
index.js:1 2022-02-11T10:36:13.916Z [ERROR] SDK - missing TURN credentials - falling back to fetch

### Have you reviewed our existing documentation?

- [X] Amazon Chime SDK for JavaScript GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] Amazon Chime SDK React Components Library GitHub [issues](https://github.com/aws/amazon-chime-sdk-component-library-react/issues)
- [X] Storybook [documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/introduction--page)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-component-library-react#readme)
- [X] How-to [documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/how-tos--page)
- [X] React SDK meeting [demo](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/meeting#readme---react-meeting-demo)

### Reproduction steps

Patient and Doctor Portal - When the both Doctor and Patient turn on the video, toggling ON and OFF is seen intermittently

### Amazon Chime SDK React Components Library version

    "amazon-chime-sdk-component-library-react": "^2.12.0",

### What browsers are you seeing the problem on?

Google Chrome

### Browser version

Version 98.0.4758.82

### Device Information

browser

### Meeting and Attendee ID Information.

ConsoleLogger.ts:82 2022-02-11T10:36:08.102Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession/SetLocalDescriptionTask error: Failed to execute 'setLocalDescription' on 'RTCPeerConnection': Failed to set local offer sdp: Failed to set local video description recv parameters for m-section with mid='1'.
log @ ConsoleLogger.ts:82
warn @ ConsoleLogger.ts:40
handleMeetingSessionStatus @ DefaultAudioVideoController.ts:1298
(anonymous) @ DefaultAudioVideoController.ts:1073
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1066
rejected @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionUpdateWithRenegotiation @ DefaultAudioVideoController.ts:1042
(anonymous) @ DefaultAudioVideoController.ts:780
perform @ DefaultSessionStateController.ts:86
update @ DefaultAudioVideoController.ts:779
resubscribe @ ReceiveVideoStreamIndexTask.ts:145
handleIndexFrame @ ReceiveVideoStreamIndexTask.ts:84
handleSignalingClientEvent @ ReceiveVideoStreamIndexTask.ts:56
sendEvent @ DefaultSignalingClient.ts:348
receiveMessage @ DefaultSignalingClient.ts:286
(anonymous) @ DefaultSignalingClient.ts:362
index.esm.js:7005 [MeetingManager audioVideoDidStart] Meeting started successfully
ConsoleLogger.ts:82 2022-02-11T10:36:11.229Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession/SetLocalDescriptionTask error: Failed to execute 'setLocalDescription' on 'RTCPeerConnection': Failed to set local offer sdp: Failed to set local video description recv parameters for m-section with mid='1'.
log @ ConsoleLogger.ts:82
warn @ ConsoleLogger.ts:40
handleMeetingSessionStatus @ DefaultAudioVideoController.ts:1298
(anonymous) @ DefaultAudioVideoController.ts:1073
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1066
rejected @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionUpdateWithRenegotiation @ DefaultAudioVideoController.ts:1042
(anonymous) @ DefaultAudioVideoController.ts:780
perform @ DefaultSessionStateController.ts:86
update @ DefaultAudioVideoController.ts:779
resubscribe @ ReceiveVideoStreamIndexTask.ts:145
handleIndexFrame @ ReceiveVideoStreamIndexTask.ts:84
handleSignalingClientEvent @ ReceiveVideoStreamIndexTask.ts:56
sendEvent @ DefaultSignalingClient.ts:348
receiveMessage @ DefaultSignalingClient.ts:286
(anonymous) @ DefaultSignalingClient.ts:362
ConsoleLogger.ts:82 2022-02-11T10:36:11.335Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession/CreateSDPTask error: Failed to execute 'createOffer' on 'RTCPeerConnection': Session error code: ERROR_CONTENT. Session error description: Failed to set local video description recv parameters for m-section with mid='1'..
log @ ConsoleLogger.ts:82
warn @ ConsoleLogger.ts:40
handleMeetingSessionStatus @ DefaultAudioVideoController.ts:1298
(anonymous) @ DefaultAudioVideoController.ts:1073
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1066
rejected @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionUpdateWithRenegotiation @ DefaultAudioVideoController.ts:1042
(anonymous) @ DefaultAudioVideoController.ts:780
perform @ DefaultSessionStateController.ts:86
update @ DefaultAudioVideoController.ts:779
resubscribe @ ReceiveVideoStreamIndexTask.ts:145
handleIndexFrame @ ReceiveVideoStreamIndexTask.ts:84
handleSignalingClientEvent @ ReceiveVideoStreamIndexTask.ts:56
sendEvent @ DefaultSignalingClient.ts:348
receiveMessage @ DefaultSignalingClient.ts:286
(anonymous) @ DefaultSignalingClient.ts:362
index.esm.js:7005 [MeetingManager audioVideoDidStart] Meeting started successfully
index.js:1 2022-02-11T10:36:13.916Z [ERROR] SDK - missing TURN credentials - falling back to fetch
console.<computed> @ index.js:1
log @ ConsoleLogger.ts:79
error @ ConsoleLogger.ts:44
(anonymous) @ ReceiveTURNCredentialsTask.ts:45
(anonymous) @ ReceiveTURNCredentialsTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/ReceiveTURNCredentialsTask.js.__awaiter @ ReceiveTURNCredentialsTask.ts:3
run @ ReceiveTURNCredentialsTask.ts:39
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ SerialGroupTask.ts:40
fulfilled @ SerialGroupTask.ts:3
Promise.then (async)
step @ SerialGroupTask.ts:3
fulfilled @ SerialGroupTask.ts:3
Promise.then (async)
step @ SerialGroupTask.ts:3
(anonymous) @ SerialGroupTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/SerialGroupTask.js.__awaiter @ SerialGroupTask.ts:3
run @ SerialGroupTask.ts:32
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ SerialGroupTask.ts:40
fulfilled @ SerialGroupTask.ts:3
Promise.then (async)
step @ SerialGroupTask.ts:3
(anonymous) @ SerialGroupTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/SerialGroupTask.js.__awaiter @ SerialGroupTask.ts:3
run @ SerialGroupTask.ts:32
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ TimeoutTask.ts:36
(anonymous) @ TimeoutTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/TimeoutTask.js.__awaiter @ TimeoutTask.ts:3
run @ TimeoutTask.ts:29
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ SerialGroupTask.ts:40
(anonymous) @ SerialGroupTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/SerialGroupTask.js.__awaiter @ SerialGroupTask.ts:3
run @ SerialGroupTask.ts:32
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ DefaultAudioVideoController.ts:1188
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionReconnect @ DefaultAudioVideoController.ts:1162
(anonymous) @ DefaultAudioVideoController.ts:1144
perform @ DefaultSessionStateController.ts:86
performDeferredAction @ DefaultSessionStateController.ts:164
perform @ DefaultSessionStateController.ts:97
(anonymous) @ DefaultAudioVideoController.ts:1203
fulfilled @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionReconnect @ DefaultAudioVideoController.ts:1162
(anonymous) @ DefaultAudioVideoController.ts:1144
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1143
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:1136
(anonymous) @ DefaultReconnectController.ts:89
(anonymous) @ TimeoutScheduler.ts:18
setTimeout (async)
start @ TimeoutScheduler.ts:16
retryWithBackoff @ DefaultReconnectController.ts:87
reconnect @ DefaultAudioVideoController.ts:1136
handleMeetingSessionStatus @ DefaultAudioVideoController.ts:1296
(anonymous) @ DefaultAudioVideoController.ts:1073
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1066
rejected @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionUpdateWithRenegotiation @ DefaultAudioVideoController.ts:1042
(anonymous) @ DefaultAudioVideoController.ts:780
perform @ DefaultSessionStateController.ts:86
update @ DefaultAudioVideoController.ts:779
resubscribe @ ReceiveVideoStreamIndexTask.ts:145
handleIndexFrame @ ReceiveVideoStreamIndexTask.ts:84
handleSignalingClientEvent @ ReceiveVideoStreamIndexTask.ts:56
sendEvent @ DefaultSignalingClient.ts:348
receiveMessage @ DefaultSignalingClient.ts:286
(anonymous) @ DefaultSignalingClient.ts:362
Show 29 more frames
Video:1 Access to fetch at 'https://2954.cell.ap-southeast-1.meetings.chime.aws/v2/turn_sessions?X-Amzn-Version=2.23.0&X-Amzn-User-Agent=chrome-90' from origin 'http://localhost:3001' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
ReceiveTURNCredentialsTask.ts:75          POST https://2954.cell.ap-southeast-1.meetings.chime.aws/v2/turn_sessions?X-Amzn-Version=2.23.0&X-Amzn-User-Agent=chrome-90 net::ERR_FAILED

### Browser console logs

ConsoleLogger.ts:82 2022-02-11T10:36:08.102Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession/SetLocalDescriptionTask error: Failed to execute 'setLocalDescription' on 'RTCPeerConnection': Failed to set local offer sdp: Failed to set local video description recv parameters for m-section with mid='1'.
log @ ConsoleLogger.ts:82
warn @ ConsoleLogger.ts:40
handleMeetingSessionStatus @ DefaultAudioVideoController.ts:1298
(anonymous) @ DefaultAudioVideoController.ts:1073
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1066
rejected @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionUpdateWithRenegotiation @ DefaultAudioVideoController.ts:1042
(anonymous) @ DefaultAudioVideoController.ts:780
perform @ DefaultSessionStateController.ts:86
update @ DefaultAudioVideoController.ts:779
resubscribe @ ReceiveVideoStreamIndexTask.ts:145
handleIndexFrame @ ReceiveVideoStreamIndexTask.ts:84
handleSignalingClientEvent @ ReceiveVideoStreamIndexTask.ts:56
sendEvent @ DefaultSignalingClient.ts:348
receiveMessage @ DefaultSignalingClient.ts:286
(anonymous) @ DefaultSignalingClient.ts:362
index.esm.js:7005 [MeetingManager audioVideoDidStart] Meeting started successfully
ConsoleLogger.ts:82 2022-02-11T10:36:11.229Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession/SetLocalDescriptionTask error: Failed to execute 'setLocalDescription' on 'RTCPeerConnection': Failed to set local offer sdp: Failed to set local video description recv parameters for m-section with mid='1'.
log @ ConsoleLogger.ts:82
warn @ ConsoleLogger.ts:40
handleMeetingSessionStatus @ DefaultAudioVideoController.ts:1298
(anonymous) @ DefaultAudioVideoController.ts:1073
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1066
rejected @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionUpdateWithRenegotiation @ DefaultAudioVideoController.ts:1042
(anonymous) @ DefaultAudioVideoController.ts:780
perform @ DefaultSessionStateController.ts:86
update @ DefaultAudioVideoController.ts:779
resubscribe @ ReceiveVideoStreamIndexTask.ts:145
handleIndexFrame @ ReceiveVideoStreamIndexTask.ts:84
handleSignalingClientEvent @ ReceiveVideoStreamIndexTask.ts:56
sendEvent @ DefaultSignalingClient.ts:348
receiveMessage @ DefaultSignalingClient.ts:286
(anonymous) @ DefaultSignalingClient.ts:362
ConsoleLogger.ts:82 2022-02-11T10:36:11.335Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms error: serial group task AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession was canceled due to subtask AudioVideoUpdate/30263450-fbe2-4fca-8cd1-64f6210b2954/4c876cbc-8f9a-8ad2-2f4a-61091383fd6b/Timeout15000ms/UpdateSession/CreateSDPTask error: Failed to execute 'createOffer' on 'RTCPeerConnection': Session error code: ERROR_CONTENT. Session error description: Failed to set local video description recv parameters for m-section with mid='1'..
log @ ConsoleLogger.ts:82
warn @ ConsoleLogger.ts:40
handleMeetingSessionStatus @ DefaultAudioVideoController.ts:1298
(anonymous) @ DefaultAudioVideoController.ts:1073
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1066
rejected @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionUpdateWithRenegotiation @ DefaultAudioVideoController.ts:1042
(anonymous) @ DefaultAudioVideoController.ts:780
perform @ DefaultSessionStateController.ts:86
update @ DefaultAudioVideoController.ts:779
resubscribe @ ReceiveVideoStreamIndexTask.ts:145
handleIndexFrame @ ReceiveVideoStreamIndexTask.ts:84
handleSignalingClientEvent @ ReceiveVideoStreamIndexTask.ts:56
sendEvent @ DefaultSignalingClient.ts:348
receiveMessage @ DefaultSignalingClient.ts:286
(anonymous) @ DefaultSignalingClient.ts:362
index.esm.js:7005 [MeetingManager audioVideoDidStart] Meeting started successfully
index.js:1 2022-02-11T10:36:13.916Z [ERROR] SDK - missing TURN credentials - falling back to fetch
console.<computed> @ index.js:1
log @ ConsoleLogger.ts:79
error @ ConsoleLogger.ts:44
(anonymous) @ ReceiveTURNCredentialsTask.ts:45
(anonymous) @ ReceiveTURNCredentialsTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/ReceiveTURNCredentialsTask.js.__awaiter @ ReceiveTURNCredentialsTask.ts:3
run @ ReceiveTURNCredentialsTask.ts:39
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ SerialGroupTask.ts:40
fulfilled @ SerialGroupTask.ts:3
Promise.then (async)
step @ SerialGroupTask.ts:3
fulfilled @ SerialGroupTask.ts:3
Promise.then (async)
step @ SerialGroupTask.ts:3
(anonymous) @ SerialGroupTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/SerialGroupTask.js.__awaiter @ SerialGroupTask.ts:3
run @ SerialGroupTask.ts:32
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ SerialGroupTask.ts:40
fulfilled @ SerialGroupTask.ts:3
Promise.then (async)
step @ SerialGroupTask.ts:3
(anonymous) @ SerialGroupTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/SerialGroupTask.js.__awaiter @ SerialGroupTask.ts:3
run @ SerialGroupTask.ts:32
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ TimeoutTask.ts:36
(anonymous) @ TimeoutTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/TimeoutTask.js.__awaiter @ TimeoutTask.ts:3
run @ TimeoutTask.ts:29
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ SerialGroupTask.ts:40
(anonymous) @ SerialGroupTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/SerialGroupTask.js.__awaiter @ SerialGroupTask.ts:3
run @ SerialGroupTask.ts:32
(anonymous) @ BaseTask.ts:63
(anonymous) @ BaseTask.ts:3
push../node_modules/amazon-chime-sdk-js/build/task/BaseTask.js.__awaiter @ BaseTask.ts:3
baseRun @ BaseTask.ts:48
(anonymous) @ DefaultAudioVideoController.ts:1188
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionReconnect @ DefaultAudioVideoController.ts:1162
(anonymous) @ DefaultAudioVideoController.ts:1144
perform @ DefaultSessionStateController.ts:86
performDeferredAction @ DefaultSessionStateController.ts:164
perform @ DefaultSessionStateController.ts:97
(anonymous) @ DefaultAudioVideoController.ts:1203
fulfilled @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionReconnect @ DefaultAudioVideoController.ts:1162
(anonymous) @ DefaultAudioVideoController.ts:1144
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1143
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:1136
(anonymous) @ DefaultReconnectController.ts:89
(anonymous) @ TimeoutScheduler.ts:18
setTimeout (async)
start @ TimeoutScheduler.ts:16
retryWithBackoff @ DefaultReconnectController.ts:87
reconnect @ DefaultAudioVideoController.ts:1136
handleMeetingSessionStatus @ DefaultAudioVideoController.ts:1296
(anonymous) @ DefaultAudioVideoController.ts:1073
perform @ DefaultSessionStateController.ts:86
(anonymous) @ DefaultAudioVideoController.ts:1066
rejected @ DefaultAudioVideoController.ts:3
Promise.then (async)
step @ DefaultAudioVideoController.ts:3
(anonymous) @ DefaultAudioVideoController.ts:3
push../node_modules/amazon-chime-sdk-js/build/audiovideocontroller/DefaultAudioVideoController.js.__awaiter @ DefaultAudioVideoController.ts:3
actionUpdateWithRenegotiation @ DefaultAudioVideoController.ts:1042
(anonymous) @ DefaultAudioVideoController.ts:780
perform @ DefaultSessionStateController.ts:86
update @ DefaultAudioVideoController.ts:779
resubscribe @ ReceiveVideoStreamIndexTask.ts:145
handleIndexFrame @ ReceiveVideoStreamIndexTask.ts:84
handleSignalingClientEvent @ ReceiveVideoStreamIndexTask.ts:56
sendEvent @ DefaultSignalingClient.ts:348
receiveMessage @ DefaultSignalingClient.ts:286
(anonymous) @ DefaultSignalingClient.ts:362
Show 29 more frames
Video:1 Access to fetch at 'https://2954.cell.ap-southeast-1.meetings.chime.aws/v2/turn_sessions?X-Amzn-Version=2.23.0&X-Amzn-User-Agent=chrome-90' from origin 'http://localhost:3001' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
ReceiveTURNCredentialsTask.ts:75          POST https://2954.cell.ap-southeast-1.meetings.chime.aws/v2/turn_sessions?X-Amzn-Version=2.23.0&X-Amzn-User-Agent=chrome-90 net::ERR_FAILED

### Add any other context about the problem here.

when 2nd attendee join video call , and start sharing his video attendee video blinking

**Comment by xuesichao** [comment]
Hi @kapilmyakal , thanks for reporting this issue. I have met the same issue recently using the Chrome with latest version. I could not reproduce this issue on Firefox and Safari and we didn't change the code related to this functionality recently. I assume this issue relates to Chrome 98 changes but need to confirm it. We will take a look at this issue and come back later.

**Comment by ltrung** [comment]
This seems like issue https://github.com/aws/amazon-chime-sdk-js/issues/1893. Could you double check if your JS SDK dependency has been updated to 2.26.1+

**Comment by kapilmyakal** [comment]
> Hi @kapilmyakal , thanks for reporting this issue. I have met the same issue recently using the Chrome with latest version. I could not reproduce this issue on Firefox and Safari and we didn't change the code related to this functionality recently. I assume this issue relates to Chrome 98 changes but need to confirm it. We will take a look at this issue and come back later.

**Comment by kapilmyakal** [comment]
> This seems like issue [aws/amazon-chime-sdk-js#1893](https://github.com/aws/amazon-chime-sdk-js/issues/1893). Could you double check if your JS SDK dependency has been updated to 2.26.1+

Hii @ltrung , we are using amazon-chime-sdk-component-library-react library

**Comment by kapilmyakal** [comment]
> t. We will take a look at this issue and come back later.
Dear @xuesichao ,
We are blocked due to this Issue, our app mostly supported only for Google Chrome Please look into this ASAP and Confirm.
in future also chrome is auto updating for all customers please look into this

**Comment by xuesichao** [comment]
@kapilmyakal I was too busy the other day, didn't have time to work on this, sorry for the late reply.
I am able to reproduce this issue when using `amazon-chime-sdk-js@2.21.1`.
After updating the `amazon-chime-sdk-js` to latest version (2.27.0), I am no longer able to reproduce it.

Could you please run `npm install amazon-chime-sdk-js@latest` in your project and see if it works?

**Comment by kapilmyakal** [comment]
> @kapilmyakal I was too busy the other day, didn't have time to work on this, sorry for the late reply. I am able to reproduce this issue when using `amazon-chime-sdk-js@2.21.1` in my `amazon-chime-sdk-component-library-react` based project. After updating the `amazon-chime-sdk-js` to latest version (2.27.0), I am no longer able to reproduce it.
> 
> Could you please run `npm install amazon-chime-sdk-js@latest` in your project and see if it works?

Thanks @xuesichao

**Comment by kapilmyakal** [comment]
> > > This seems like issue [aws/amazon-chime-sdk-js#1893](https://github.com/aws/amazon-chime-sdk-js/issues/1893). Could you double check if your JS SDK dependency has been updated to 2.26.1+
> > 
> > 
> > Hii @ltrung , we are using amazon-chime-sdk-component-library-react library
> 
> @kapilmyakal So amazon-chime-sdk-component-library-react has a dependency on the main amazon-chime-js-sdk which is the root cause of this. Could you confirm your amazon-chime-sdk-js version first?
your solution worked 
Thanks @ltrung

**Comment by ameriles** [comment]
I'm experiencing a similar issue, but in my case, I'm using amazon-chime-sdk-js v2.28.0.
Using Safari, when I toggle the camera off, the TileGrid re-renders.
I captured browser WARN logs for turning off the camera.

**When I toggle off my camera using using Safari 14.1.1** 
`[Warning] 2022-03-11T20:31:53.765Z [WARN] SDK - Could not find MID for stream ID: 0 (bundle.js, line 38812)
[Warning] 2022-03-11T20:31:54.086Z [WARN] SDK - will retry due to status code SignalingInternalServerError (bundle.js, line 38812)
[Warning] 2022-03-11T20:31:54.089Z [WARN] SDK - will retry due to status code TaskFailed and error: serial group task AudioVideoUpdate/3965ccdd-bc1a-486d-b865-b76f43cd0706/98138e2d-8abe-4748-cedb-7ef1b07681bd was canceled due to subtask AudioVideoUpdate/3965ccdd-bc1a-486d-b865-b76f43cd0706/98138e2d-8abe-4748-cedb-7ef1b07681bd/Timeout15000ms error: serial group task AudioVideoUpdate/3965ccdd-bc1a-486d-b865-b76f43cd0706/98138e2d-8abe-4748-cedb-7ef1b07681bd/Timeout15000ms/UpdateSession was canceled due to subtask AudioVideoUpdate/3965ccdd-bc1a-486d-b865-b76f43cd0706/98138e2d-8abe-4748-cedb-7ef1b07681bd/Timeout15000ms/UpdateSession/SubscribeAndReceiveSubscribeAckTask error: null is not an object (evaluating 'subscribeAckFrame.compressedSdpAnswer') (bundle.js, line 38812)
[Warning] 2022-03-11T20:31:57.066Z [WARN] SDK - Could not find MID for stream ID: 0 (bundle.js, line 38812)
[Warning] 2022-03-11T20:31:57.303Z [WARN] SDK - no attendee found for track v_3 (bundle.js, line 38812)
[Warning] 2022-03-11T20:31:57.304Z [WARN] SDK - stream not found for tile=4 track=v_3 (bundle.js, line 38812)
[Warning] 2022-03-11T20:31:57.304Z [WARN] SDK - no external user id found for track v_3 (bundle.js, line 38812)
[Warning] 2022-03-11T20:31:58.098Z [WARN] SDK - no stream found for tile=4 (bundle.js, line 38812)
[Log] [MeetingManager audioVideoDidStart] Meeting started successfully (bundle.js, line 23772)
[Warning] 2022-03-11T20:31:59.881Z [WARN] SDK - Could not find MID for stream ID: 0 (bundle.js, line 38812)
[Warning] 2022-03-11T20:32:00.120Z [WARN] SDK - no attendee found for track v_3 (bundle.js, line 38812)
[Warning] 2022-03-11T20:32:00.120Z [WARN] SDK - stream not found for tile=6 track=v_3 (bundle.js, line 38812)
[Warning] 2022-03-11T20:32:00.120Z [WARN] SDK - no external user id found for track v_3 (bundle.js, line 38812)
[Log] [MeetingManager audioVideoDidStart] Meeting started successfully (bundle.js, line 23772)`


**When I toggle off my camera using but using Google Chrome instead (98.0.4758.109 (arm64))**
`2022-03-11T20:34:11.209Z [WARN] SDK - Could not find MID for stream ID: 0`

**Comment by xuesichao** [comment]
Hi @ameriles, I reopened the issue. Could you please further describe what do you mean by:
>Using Safari, when I toggle the camera off, the TileGrid re-renders

What does re-renders mean exactly and what is the observation from local attendee and remote attendee?

**Comment by ameriles** [comment]
> Hi @ameriles, I reopened the issue. Could you please further describe what do you mean by:
> 
> > Using Safari, when I toggle the camera off, the TileGrid re-renders
> 
> What does re-renders mean exactly and what is the observation from local attendee and remote attendee?

Yes sure, let me elaborate. 

When I toggle the local camera off, the TileGrid stops displaying the remote videos for a few seconds, then the remote tile videos appears again. The LocalVideo is not showing (this is expected).
If I enable the local video again, the local video tile shows as a black square.

**Comment by ltrung** [comment]
@ameriles I will close this for now since the original issue has been resolved and this issue is with Safari. Can you open a new issue for your case and upload a complete console log so we can have more information to troubleshoot? From the log you uploaded, it seems reconnection happend which causes remote videos to drop.

**Comment by ameriles** [comment]
> @ameriles I will close this for now since the original issue has been resolved and this issue is with Safari. Can you open a new issue for your case and upload a complete console log so we can have more information to troubleshoot? From the log you uploaded, it seems reconnection happend which causes remote videos to drop.

Thank @ltrung. I created a new issue and posted the full logs there.
